src/data/wallets/wallet-filters.ts

```diff
@@ -131,8 +131,9 @@ const walletFilterData: { [key: string]: WalletFilter } = {
     filterKey: "erc_20_support",
   },
   eip_1559_support: {
```

Comment:
If we remove mention of EIP 1559, should we update this `eip_1559_support` key, e.g. to `type_2_tx_support`?

Reply:
Been looking at this code again and I think there should be a refactor on the filters, I think for myself keeping it as 
`eip_1559_support` helps as it is clear what it is as a variable, but open to suggestions